packages/components/Shimmer/FURNShimmer.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+  s.name             = 'FURNShimmer'
+  s.version          = '0.1.0'
+  s.summary          = 'Fluent UI React Native iOS Shimmer Control'
+  s.homepage         = "https://www.microsoft.com/design/fluent/#/"
+  s.license          = { :type => 'MIT', :file => 'LICENSE' }
+  s.author           = { "Microsoft" => "fluentuinativeowners@microsoft.com"}
+  s.source       = { :git => "https://github.com/microsoft/fluentui-react-native.git", :tag => "#{s.version}" }
+  s.swift_version = "5.0"
+  s.module_name = 'FURNShimmer'
+
+  s.ios.deployment_target = "11.0"
+  s.ios.source_files   = "ios/*.{swift,h,m}"
+
+  s.dependency 'React'
+  s.dependency 'MicrosoftFluentUI', '~> 0.1.0'
```

Comment:
lets use the latest

Reply:
From https://guides.cocoapods.org/using/the-podfile.html

====
In addition to the logic operators CocoaPods has an optimistic operator ~>:

'~> 0.1.2' Version 0.1.2 and the versions up to 0.2, not including 0.2 and higher
'~> 0.1' Version 0.1 and the versions up to 1.0, not including 1.0 and higher
'~> 0' Version 0 and the versions up to 1.0, not including 1.0 and higher
=====

So this should be taking our latest, but will not take the 1.0.0 release whenever that happens, I believe